src/wallet/models.py

```diff
@@ -93,3 +100,7 @@ def countable(self):
         else:
             return self.trx_status in [enums.TrxStatus.PENDING,
                                        enums.TrxStatus.FINALIZED]
+
+    class Meta:
+        verbose_name = 'Transaction'
```

Comment:
The verbose names need to be internationalised (check all the instances in this file).